hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMDirectoriesPurgeRequestAndResponse.java

```diff
@@ -52,6 +53,13 @@ public class TestOMDirectoriesPurgeRequestAndResponse extends TestOMKeyRequest {
    * Creates volume, bucket and key entries and adds to OM DB and then
    * deletes these keys to move them to deletedKeys table.
    */
+  @Before
+  public void cleanup() throws Exception {
+    String bucketKey = omMetadataManager.getBucketKey(volumeName,
+        bucketName);
+    omMetadataManager.getBucketTable().delete(bucketKey);
+  }
+  
```

Comment:
Appears to be unnecessary.  Remove?  Tests pass without.  Also `bucketKey` appears to never be in `bucketTable` when called.

Reply:
Thanks @sumitagrawl for your updates.  Will look at it shortly.